### DISCUSSION TITLE
Interpolate variable containing mail does not work?

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#295] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v10.6.0

### Description

Hi

I have a variable that contains an email, for example in the `DEV.env` file I have:

`MAIL_RECIPIENT=mail@example.com`

But when I try to deploy I receive an error from the pipeline. The message is:

```
error converting YAML to JSON: yaml: line 16: did not find expected alphabetic or numeric character
```

Does mlp support interpolation for email formatted strings? 
Thanks

**Comment by malta895** [comment, marked as answer]
Hi @flower-of-the-bridges, have you tried enclosing the email in quotes? 
`MAIL_RECIPIENT='mail@example.com'`

**Reply by flower-of-the-bridges** [reply]
Hi @malta895 thank you very much :heart: it works
